kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/LogReplay.java

```diff
@@ -409,11 +426,13 @@ private Map<String, DomainMetadata> loadDomainMetadataMap(Engine engine) {
    * #loadTableProtocolAndMetadata}.
    *
    * @param engine The engine used to process the log files.
+   * @param minLogVersion the minimum log version to read
```

Comment:
nit - inclusive, exclusive? be specific

Reply:
also not sure if the docs can be more specific to explain what is returned w.r.t the minVersion provided (i.e. if the minVersion is a log file, it'll just be a map of the latest domain metadata for each domain for the version range (and can be combined with the state from the previous version to get a full state), what if the minVersion is a checkpoint?) What if there is no minVersion? Then it's the entire state right?